API_register/views.py

```diff
@@ -1,16 +1,29 @@
-from django.shortcuts import render
 from rest_framework import generics
-from django.http import Http404
+from django_filters.rest_framework import DjangoFilterBackend
+from django_filters.widgets import RangeWidget
+import django_filters
```

Comment:
Please read and follow pep8 https://www.python.org/dev/peps/pep-0008/#imports.

You might checkout `pep8` and `isort` python tools